backend/authentication/views.py

```diff
@@ -69,13 +85,14 @@ def create(self, request, *args, **kwargs):
                                                        "stamm": serializers.data.get('stamm'),
                                                        "group": serializers.data.get('group'),
                                                    }}, exist_ok=False)
-            return Response({'status': 'ok', 'user': new_user}, status=status.HTTP_200_OK)
-        except KeycloakGetError:
-            return Response({'status': 'failed', 'error': 'user already exists'}, status=status.HTTP_400_BAD_REQUEST)
+            keycloak_admin.group_user_add(user_id=new_user, group_id=group_id)
```

Comment:
Hmm problem here is that if we add user to group without approval, he/she has access to everything even without approval. I think we need to either disable the users in keycloak or store in separate database before adding :(